### ISSUE TITLE
Inaccurate Answers to a Quiz

### ISSUE THREAD

**Issue by aoool** [udacity/sdc-issue-reports#554]

**Comment by aoool** [comment]
Lesson "MiniFlow", topic "Gradient Descent Part 1". Multiple answers can be given to a quiz, however, it is said that only one answer can be accepted. Other answers can be accepted too in theory, but for that cases, the following motivations are given: "... it is highly unlikely ...", "... in practice, you will never see this ...".

I think this is inaccurate and all the theoretically possible answers should be accepted since neural networks is a math concept.

**Comment by domluna** [comment]
Updated the quiz to have multiple answers and replaced momentum with learning rate. Thanks for the feedback!
